eaf.el

```diff
@@ -300,6 +303,59 @@ Try not to modify this alist directly. Use `eaf-setq' to modify instead."
 Any new app should add the its name and the corresponding
 keybinding variable to this list.")
 
+
+(defvar eaf--browser-current-url nil)
+(defvar-local eaf--full-title nil)
+
+(defun eaf--bookmark-make-record ()
+  "Create a eaf bookmark.
+
+The bookmark will try to recreate eaf buffer session.
+For now only eaf browser app is supported."
+  (cond ((equal eaf--buffer-app-name "browser")
+         ;; set `eaf--browser-current-url'
+         (eaf-call "execute_function" eaf--buffer-id
+                   "set_bookmark")
+         (let ((bookmark `((handler . eaf--bookmark-restore)
+                           (eaf-app . "browser")
+                           (defaults . ,(list eaf--full-title))
+                           ;; not a filename but this shows url in bookmark-list
+                           ;; which is nice
+                           (filename . ,eaf--browser-current-url))))
```

Comment:
Change eaf--browser-current-url to eaf--bookmark-link

Reply:
Done